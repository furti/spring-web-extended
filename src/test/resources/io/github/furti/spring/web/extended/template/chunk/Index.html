<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
	<p>Some static text that may contain some umlauts as ä, ü or ö.
	<p>
	<p>And a {.simple.simple} tag</p>
	<p data-attr="{.static.}">There should be some "static text" in
		the attribute ;)</p>
	<h3>Hopefully this arrow ({.simple.-->}) gets escaped :)</h3>
	<h3
		other-attr="{.simple."escaped"}"></h3>
	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla
		adipiscing {.simple.sapien} nec nunc faucibus mollis. Donec egestas
		ipsum quis neque lobortis, non commodo erat vulputate. Sed magna
		mauris, fermentum a rhoncus eget, volutpat vitae nisl. Nullam et lacus
		nisi. Cum sociis natoque penatibus et magnis dis parturient montes,
		nascetur ridiculus mus. Curabitur euismod dolor id quam volutpat
		cursus. Duis sapien augue, convallis vel vestibulum vel, tincidunt ac
		felis. Phasellus in felis vulputate, posuere dui a, condimentum diam.
		Cras non lorem et orci sagittis ultrices. Sed rutrum libero non felis
		consequat malesuada. Fusce ultrices ante nunc, a vehicula quam
		ultricies eget. {.simple.Cras} cursus lacus ultrices, porttitor orci
		eu, semper lorem. Phasellus tincidunt enim mi. Sed et justo blandit,
		ultrices nisl eget, ornare tortor.</p>
	<p>Lorem ipsum dolor sit amet, {.simple.consectetur} adipiscing
		elit. Vestibulum fringilla mauris nec lacus dictum, cursus volutpat
		nulla accumsan. Morbi eros justo, ullamcorper bibendum nibh a, aliquam
		aliquam diam. Cras congue est et libero {.simple.lobortis}, non
		accumsan quam molestie. Nulla pulvinar nunc at lectus sagittis semper
		sed {.simple.vitae} risus. Quisque hendrerit nisl nec rutrum sodales.
		Donec cursus vulputate odio, iaculis vestibulum elit. Maecenas odio
		turpis, interdum in tempor vitae, tempor ac {.simple.felis}. Aenean
		vel suscipit ligula. Sed ultrices condimentum orci sed pellentesque.
		Pellentesque malesuada arcu a quam consequat semper. Aenean
		{.simple.volutpat} dapibus augue, eu feugiat ante tincidunt eu.
		Vestibulum pretium lacus et erat imperdiet malesuada. Suspendisse
		rhoncus sem {.simple.tristique} libero pretium, in auctor eros
		dignissim.</p>
	<p>Nullam scelerisque tincidunt dapibus. Vestibulum luctus
		scelerisque velit. Nulla volutpat purus odio, vel aliquam mauris
		lobortis at. Mauris at lacus {.simple.vitae} felis volutpat
		vestibulum. Nulla facilisi. Sed vel semper nunc. Sed eleifend a dolor
		id rhoncus. Fusce metus nisi, auctor ac lobortis vitae, dapibus vel
		quam. Suspendisse auctor euismod libero, eu iaculis mauris convallis
		vehicula.</p>
	<p>Praesent id hendrerit nulla, at varius elit. Aenean viverra,
		magna a ultrices faucibus, enim purus {.simple.molestie} augue, nec
		commodo libero mi sit amet arcu. Morbi vitae ornare lacus. Aliquam
		sollicitudin condimentum ipsum, sit amet consequat lectus accumsan
		sed. Aenean eget facilisis libero, in {.simple.sodales} turpis. Donec
		id dolor dolor. Curabitur et elit vitae purus volutpat sollicitudin
		vitae id lectus.</p>
	<p>Fusce aliquam urna enim. In mattis ligula eget odio sagittis,
		vitae aliquet leo sagittis. Donec ac blandit neque, id
		{.simple.blandit} metus. Integer semper non justo eget laoreet. {.simple.Proin}
		rutrum nisl et tortor egestas, in ornare sem porttitor. Phasellus
		sodales interdum purus, et sollicitudin risus aliquet sed. Cras ac
		iaculis erat, non lacinia sapien. Integer non faucibus dolor, semper
		lobortis nulla. Maecenas non {.simple.eros} sed turpis volutpat
		luctus. Maecenas aliquet nunc at accumsan fringilla.</p>
</body>
</html>