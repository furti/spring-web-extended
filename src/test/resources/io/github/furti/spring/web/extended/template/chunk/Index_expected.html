<!DOCTYPE html>
<html>
<head>
<title>Test</title>
</head>
<body>
	<p>Some static text that may contain some umlauts as ä, ü or ö.
	<p>
	<p>And a simple tag</p>
	<p data-attr="static text">There should be some "static text" in
		the attribute ;)</p>
	<h3>Hopefully this arrow (--&gt;) gets escaped :)</h3>
	<h3
		other-attr="&#34;escaped&#34;"></h3>
	<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla
		adipiscing sapien nec nunc faucibus mollis. Donec egestas
		ipsum quis neque lobortis, non commodo erat vulputate. Sed magna
		mauris, fermentum a rhoncus eget, volutpat vitae nisl. Nullam et lacus
		nisi. Cum sociis natoque penatibus et magnis dis parturient montes,
		nascetur ridiculus mus. Curabitur euismod dolor id quam volutpat
		cursus. Duis sapien augue, convallis vel vestibulum vel, tincidunt ac
		felis. Phasellus in felis vulputate, posuere dui a, condimentum diam.
		Cras non lorem et orci sagittis ultrices. Sed rutrum libero non felis
		consequat malesuada. Fusce ultrices ante nunc, a vehicula quam
		ultricies eget. Cras cursus lacus ultrices, porttitor orci
		eu, semper lorem. Phasellus tincidunt enim mi. Sed et justo blandit,
		ultrices nisl eget, ornare tortor.</p>
	<p>Lorem ipsum dolor sit amet, consectetur adipiscing
		elit. Vestibulum fringilla mauris nec lacus dictum, cursus volutpat
		nulla accumsan. Morbi eros justo, ullamcorper bibendum nibh a, aliquam
		aliquam diam. Cras congue est et libero lobortis, non
		accumsan quam molestie. Nulla pulvinar nunc at lectus sagittis semper
		sed vitae risus. Quisque hendrerit nisl nec rutrum sodales.
		Donec cursus vulputate odio, iaculis vestibulum elit. Maecenas odio
		turpis, interdum in tempor vitae, tempor ac felis. Aenean
		vel suscipit ligula. Sed ultrices condimentum orci sed pellentesque.
		Pellentesque malesuada arcu a quam consequat semper. Aenean
		volutpat dapibus augue, eu feugiat ante tincidunt eu.
		Vestibulum pretium lacus et erat imperdiet malesuada. Suspendisse
		rhoncus sem tristique libero pretium, in auctor eros
		dignissim.</p>
	<p>Nullam scelerisque tincidunt dapibus. Vestibulum luctus
		scelerisque velit. Nulla volutpat purus odio, vel aliquam mauris
		lobortis at. Mauris at lacus vitae felis volutpat
		vestibulum. Nulla facilisi. Sed vel semper nunc. Sed eleifend a dolor
		id rhoncus. Fusce metus nisi, auctor ac lobortis vitae, dapibus vel
		quam. Suspendisse auctor euismod libero, eu iaculis mauris convallis
		vehicula.</p>
	<p>Praesent id hendrerit nulla, at varius elit. Aenean viverra,
		magna a ultrices faucibus, enim purus molestie augue, nec
		commodo libero mi sit amet arcu. Morbi vitae ornare lacus. Aliquam
		sollicitudin condimentum ipsum, sit amet consequat lectus accumsan
		sed. Aenean eget facilisis libero, in sodales turpis. Donec
		id dolor dolor. Curabitur et elit vitae purus volutpat sollicitudin
		vitae id lectus.</p>
	<p>Fusce aliquam urna enim. In mattis ligula eget odio sagittis,
		vitae aliquet leo sagittis. Donec ac blandit neque, id
		blandit metus. Integer semper non justo eget laoreet. Proin
		rutrum nisl et tortor egestas, in ornare sem porttitor. Phasellus
		sodales interdum purus, et sollicitudin risus aliquet sed. Cras ac
		iaculis erat, non lacinia sapien. Integer non faucibus dolor, semper
		lobortis nulla. Maecenas non eros sed turpis volutpat
		luctus. Maecenas aliquet nunc at accumsan fringilla.</p>
</body>
</html>